### ISSUE TITLE
Learn verbs which incorporate prepositional information

### ISSUE THREAD

**Issue by rgabbard** [isi-vista/adam#45]
* enter = "go into"
* approach = "go toward"
* cross = "go across"

(Jackendoff 1983, 1990)

**Comment by gabbard** [comment]
@paynesa : Can you check if any of these ended up getting placed in the phase 2 curriculum?

**Comment by paynesa** [comment]
@gabbard I don't think they have yet, should I add them?

**Comment by LichMaster98** [comment]
@paynesa @gabbard 
Into was ruled out of scope: https://github.com/isi-vista/adam/issues/765
Towards was added in: https://github.com/isi-vista/adam/pull/767

**Comment by gabbard** [comment]
@LichMaster98 : right, but we didn't "lexicalize" any of these by including e.g. `enter`/`approach`/`cross` in our curriculum?

**Comment by gabbard** [comment]
I we didn't add them I am going to shift this off the Phase 2 milestone.

**Comment by LichMaster98** [comment]
@gabbard We don't lexicalize any of these verbs.
